src/main/java/com/paskar/email/application/service/EmailRepositoryJdbcImpl.java

```diff
@@ -0,0 +1,67 @@
+package com.paskar.email.application.service;
+
+import com.paskar.email.application.mapper.EmailMapper;
+import com.paskar.email.application.model.Email;
+import com.paskar.email.application.repositiory.EmailRepositoryJdbc;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.jdbc.core.JdbcTemplate;
+import org.springframework.stereotype.Component;
+
+import java.time.LocalDate;
+import java.util.ArrayList;
+import java.util.List;
+
+@Component
+public class EmailRepositoryJdbcImpl implements EmailRepositoryJdbc {
```

Comment:
This is not correct approach.
You should provide single interface and multiple implementations.

```
interface EmailRepository {
    List<Email> findAll();
    // ...
}
```

```
class FileRepository implements EmailRepository {
    @Override
    public List<Email> findAll() {
        new FileReader(...).readLine();
        // ...
    }

    // ...
}
```

```
class JdbcRepository implements EmailRepository {
    @Override
    public List<Email> findAll() {
        jdbcTemplate.query(...);
        // ...
    }

    // ...
}
```